src/components/Transcript/TranscriptMenu/TranscriptSelector.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import TranscriptDownloader from './TranscriptDownloader';
+
```

Comment:
Might want to pull in the `prop-types` package just to make it more clear to other devs, which props are expected for the component?

Reply:
Absolutely!